packages/keycloak-user-management/src/components/CreateEditUser/index.tsx

```diff
@@ -58,16 +68,39 @@ export const defaultEditUserProps: EditUserProps = {
 
 const CreateEditUser: React.FC<CreateEditPropTypes> = (props: CreateEditPropTypes) => {
   const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [practitioner, setPractitioner] = React.useState<Practitioner | undefined>(undefined);
   const {
     keycloakUser,
     accessToken,
     keycloakBaseURL,
+    opensrpBaseURL,
     serviceClass,
+    opensrpServiceClass,
     fetchKeycloakUsersCreator,
   } = props;
   const userId = props.match.params[ROUTE_PARAM_USER_ID];
   const initialValues = keycloakUser ? keycloakUser : defaultInitialValues;
 
+  const loadPractitioner = async () => {
+    if (userId && practitioner === undefined) {
+      setIsLoading(true);
+      const serve = new opensrpServiceClass(
+        accessToken,
+        opensrpBaseURL,
+        OPENSRP_CREATE_PRACTITIONER_ENDPOINT
+      );
+      serve
+        .read(userId)
+        .then((response: Practitioner) => {
+          setIsLoading(false);
+          setPractitioner(response);
+        })
+        .catch((_: Error) => {
+          sendErrorNotification(ERROR_OCCURED);
```

Comment:
In the event we have an error from the API, seems there will be an unending loader being shown. I think we should create an issue to ensure that is handled in the future

Reply:
Nice catch @kahummer